test/scraper/schleswig_holstein_landtag_scraper_overview_test.rb

```diff
@@ -58,6 +58,53 @@ def setup
 
   test 'get pdf url' do
     assert_equal 'http://www.landtag.ltsh.de/infothek/wahl18/drucks/2500/drucksache-18-2537.pdf', @scraper.extract_url(@blocks[15])
+    assert_equal 'http://www.landtag.ltsh.de/infothek/wahl18/drucks/2500/drucksache-18-2540.pdf', @scraper.extract_url(@blocks[0])
+  end
+
+  test 'get major paper' do
+    html = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/schleswigholstein_landtag_scraper_major.html')).force_encoding("WINDOWS-1252"))
+    table = @scraper.extract_table html
+    blocks = @scraper.extract_blocks table
+    paper = @scraper.extract_major_paper blocks[0]
+    assert_equal({
+      legislative_term: '17',
+      full_reference: '17/2295',
+      reference: '2295',
+      doctype: Paper::DOCTYPE_MAJOR_INTERPELLATION,
+      title: 'Kinder und Jugendliche mit Migrationshintergrund im Bildungssystem Schleswig-Holsteins',
+      url: 'http://www.landtag.ltsh.de/infothek/wahl17/drucks/2200/drucksache-17-2295.pdf',
+      published_at: Date.parse('15.02.2012'),
+      is_answer: true,
+      answerers: { ministries: ["Landesregierung"] }
+    }, paper)
+  end
+
+  test 'block is minor or major' do
+    assert_equal false, @scraper.major?(@blocks[0])
+
+    html = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/schleswigholstein_landtag_scraper_major.html')
+                          ).force_encoding("WINDOWS-1252"))
+    table = @scraper.extract_table html
+    block = @scraper.extract_blocks table
+    assert_equal true, @scraper.major?(block)
+
+  end
+
+  test 'update details' do
+    html = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/schleswigholstein_landtag_scraper_major_detail.html')
+                          ).force_encoding("WINDOWS-1252"))
+    block = @scraper.extract_detail_block(html)
+    assert_equal false, block.nil?
+    line = @scraper.extract_originator_line block
+    assert_equal 'Anke Erdmann (BÜNDNIS 90/DIE GRÜNEN) 07.10.2011 Drucksache', line
+    originators = NamePartyExtractor.new(line).extract
+
+    assert_equal({
+                   :people=>["Anke Erdmann"],
+                   :parties=>["BÜNDNIS 90/DIE GRÜNEN"]
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.